src/types.ts

```diff
@@ -86,7 +86,7 @@ export interface Options {
    *
    * @default 'vue3'
    */
-  transformer?: 'vue3' | 'vue2'
+  transformer?: 'vue2' | 'vue3' | 'svelte315'
```

Comment:
What's 315 here?

Reply:
Svelte support starts with svelte version 3.15.

Reply:
wait no, it's actually 3.13. Changing it

Reply:
I think you can write it to readme, in here I preferred svelte3.

Reply:
How about just Svelte3? Would this approach not compatible with Svelte under 3.13?

Reply:
It's not compatible with svelte under 3.13, unfortunately.
I just realized that there may be an easy workaround. Will try it later today.

Reply:
edit: will try later, hopefully tomorrow